### PR TITLE
fix property of undefined split

### PR DESCRIPTION
**Pull request by MariaAvramescu26**
trying to fix property of undefined split.

### CODE REVIEW

**Comment by MariaAvramescu26** [comment]
Main modifications:

- Consistent Quoting:Ensured consistent use of double quotes throughout the code.
- Corrected Access token generated Message:Fixed the message to ensure it's printed correctly.
- Updated Comments:Adjusted some comments for clarity.
- General Cleanup:Removed unnecessary line breaks and spaces.

I'm not certain if this will resolve the issue because, although the error no longer occurs when I test it with my Codecept code, unfortunately, the reports are still not being generated. This leaves me uncertain whether the problem persists in the plugin or within my test itself.
I'll leave it to your decision whether you'd like to merge this code. Going forward, I can then attempt to use this updated version of the plugin in my tests. Alternatively, I'm open to scheduling a video call to collaborate and ensure everything is functioning correctly. Thank you.

File: package.json
Comment (medtoure18) [inline package.json:1]: Why adding codeceptjs version ?

**Review by medtoure18 — COMMENTED** [review]
just on comment otherwise OK

**Comment by medtoure18** [comment]
@MariaAvramescu26 i published your branch on a beta version on npm:
https://www.npmjs.com/package/codeceptjs-xray-helper/v/3.0.1-beta.0
You can try, if it's ok for you, i will merge

**Comment by MariaAvramescu26** [comment]
hei Med, can you prepare me a beta with this last two changes I added? thanks a lot!

**Comment by medtoure18** [comment]
HI Maria,
it's done:
https://www.npmjs.com/package/codeceptjs-xray-helper/v/3.0.1-beta.2

**Comment by MariaAvramescu26** [comment]
Hi Med,
A new issue has occured with an undefined split on line 99. It seems like an endless story XD. If you don't mind, could you create a beta version with this updated code? If you feel it's not worth the effort, I completely understand. 
Please let me know, and thanks for your patience.
Have a great one!

**Comment by medtoure18** [comment]
Hi @MariaAvramescu26,
version codeceptjs-xray-helper@3.0.1-beta.3 published.
If you want i can give acces to the npm project if you have an npm account so you can piblish version yourself:
 all you have to do if you have access to the project :
 - be on on package.json: 
 - add beta to version on package.json and run :
 - npm login
 - npm publish --tag beta

**Comment by MariaAvramescu26** [comment]
I never published a version myself but I will be happy to try it out. I got my NPM account created! Let me know when I could give it a try, thanks Med!

**Comment by medtoure18** [comment]
@MariaAvramescu26 give me your npm user login.
i published beta4

**Comment by MariaAvramescu26** [comment]
my npm user login is mary123456666 email maria.avramescu123@gmail.com

**Comment by MariaAvramescu26** [comment]
Thanks Med, those days I will be a bit busy with work, but I will try it out asap! have a great weekend!

**Comment by MariaAvramescu26** [comment]
Hello Med,
It appears that everything is finally working as expected, so I am merging the final result. 

**By the way, I'm still a bit unclear on how to link an existing test case in my Xray repository with the test execution. Could you please provide some guidance? Thank you a milion!**

For example, I've copied my test case from the Xray repository:

@smoke
Scenario: User can successfully create a support ticket
    Given I am logged in 
    And I am on the website in help-center section
    And I create a support ticket
    Then the support ticket is properly created 

And when i run  npx codeceptjs run --steps --grep "@smoke" i got the following result:
```
examples:[]
examples:[]
SEND TO XRAY => {"info" : {"summary" : "Execution of automated tests for release ",  "startDate" : "2024-03-12T09:39:17+01:00", "finishDate" :"2024-03-12T09:39:17+01:00","revision": "","description" : "Results of test execution ", "testEnvironments": ["browser:chrome", "windows"]},"tests" : []}
```

**Comment by medtoure18** [comment]
Hi Maria?
You have to add your jira issue id as tag of your scenario.

If your jira project test issue is PRX-123.
So your scénario has to be:

@smoke @PRX-123
Scenario: User can successfully create a support ticket
Given I am logged in
And I am on the website in help-center section
And I create a support ticket
Then the support ticket is properly created

**Comment by MariaAvramescu26** [comment]
Thanks Med, so kind of you!
I added: 
@PRE1 @EN-45
 Scenario: Navigate to the home page by clicking on the logo
       Given I am on the website in help-center section 
       When I click on the logo in the top left corner
       Then the home page should be loaded

When I run command npx codeceptjs run --steps --grep "@PRE1" , I get this: 
`examples:[]
SEND TO XRAY => { "testExecutionKey": "PRE1","info" : {"startDate" : "2024-03-13T10:58:54+01:00", "finishDate" :"2024-03-13T10:58:54+01:00","revision": "001","description" : "Results of test execution ", "testEnvironments": ["browser:chrome", "windows"]},"tests" : []}`

There is no test execution created. Any idea why this could be?
Thanks a lot!

**Comment by medtoure18** [comment]
I think it does not call jira side. Maybe there is a part of the that it
does not excute.

**Pull request merged**